tests/Behat/UserGraphQLContext/UserGraphQLContext.php

```
@@ -267,31 +265,315 @@
-            true
-        )['data'][$this->queryName]['edges'];
+        $content = $this->getPageContent();
+        $data = json_decode($content, true)['data'][$this->queryName];
 
-        Assert::assertIsArray($userData);
-        foreach ($userData as $user) {
-            foreach ($this->responseContent as $item) {
-                Assert::assertArrayHasKey($item, $user['node']);
-            }
-        }
+        Assert::assertArrayHasKey('edges', $data);
```

PHPMD (2.15.0)

Avoid using static access to class '\PHPUnit\Framework\Assert' in method 'collectionOfUsersShouldBeReturned'. (Clean Code Rules)

(StaticAccess)